server.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express")
+
+const app = express()
+
+app.get("/ping",(req,res) => {
+    res.send("Hi This is Waran")
+})
+
+app.listen(3000,() => {
```

Comment:
Consider making the server port configurable through environment variables. This can be done using `process.env.PORT || 3000`. This allows flexibility in deployment environments where the port might be set by the environment. [important]